### PR TITLE
Return settlement fees with settlement history

### PR DESCRIPTION
**Pull request by mdomasevicius**

### CODE REVIEW

**Review by vkuznecovas — APPROVED** [review]

File: tequilapi/contract/transactor.go [hunk 1]
```diff
@@ -153,6 +157,7 @@ func NewSettlementDTO(settlement pingpong.SettlementHistoryEntry) SettlementDTO
 		Beneficiary:    settlement.Beneficiary.Hex(),
 		Amount:         settlement.Amount.Uint64(),
```
Comment (vkuznecovas) [inline tequilapi/contract/transactor.go:158]: this is a bug, you'll exceed the uints rather quick

File: tequilapi/contract/transactor.go [hunk 2]
```diff
@@ -153,6 +157,7 @@ func NewSettlementDTO(settlement pingpong.SettlementHistoryEntry) SettlementDTO
...
 		SettledAt:      settlement.Time.Format(time.RFC3339),
+		Fees:           fees.Uint64(),
```
Comment (vkuznecovas) [inline tequilapi/contract/transactor.go:160]: this is a bug, you'll exceed the uints rather quick

**Review by vkuznecovas — CHANGES_REQUESTED** [review]

File: tequilapi/contract/transactor.go (hunk 1 above)
Comment (vkuznecovas) [reply tequilapi/contract/transactor.go:158]: 19 myst and you overflow

File: tequilapi/contract/custom_json_types.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package contract
+
+import (
+	"fmt"
+	"math/big"
+)
+
+type BigInt struct {
+	big.Int
```
Comment (tomasmik) [inline tequilapi/contract/custom_json_types.go:9]: If you use `*big.Int` you'll lose on some of the cool things it [implements](https://golang.org/pkg/math/big/): https://play.golang.org/p/0RrQwVdgGlb

File: tequilapi/contract/custom_json_types.go [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	return []byte(b.String()), nil
+}
+
+func (b *BigInt) UnmarshalJSON(raw []byte) error {
+	if string(raw) == "null" {
+		return nil
+	}
+
+	var t big.Int
+	_, ok := t.SetString(string(raw), 10)
```
Comment (tomasmik) [inline tequilapi/contract/custom_json_types.go:26]: You could also do something like this `t, ok := new(big.Int).SetString(..)`.

**Review by vkuznecovas — APPROVED** [review]

**Review by tomasmik — APPROVED** [review]

**Pull request merged**